tests/reals/real_timer_renames.h

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#define timer_create          real_timer_create
-#define timer_start           real_timer_start
-#define timer_get_elapsed     real_timer_get_elapsed
-#define timer_get_elapsed_ms  real_timer_get_elapsed_ms
-#define timer_destroy         real_timer_destroy
+#define timer_create            real_timer_create
+#define timer_start             real_timer_start
+#define timer_get_elapsed       real_timer_get_elapsed
+#define timer_get_elapsed_ms    real_timer_get_elapsed_ms
+#define timer_destroy           real_timer_destroy
+#define g_timer_get_elapsed_ms  real_g_timer_get_elapsed_ms
```

Comment:
>g_timer_get_elapsed_ms [](start = 8, length = 22)

Small ask: timer_global_get_elapsed_ms pretty please. Simply because g_ in front makes my brain immediately think of a global *variable*.

Reply:
renamed